app/javascript/components/region-form/index.jsx

```diff
@@ -0,0 +1,67 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import { Grid } from 'patternfly-react';
+import createSchema from './region-form.schema';
+import MiqFormRenderer from '../../forms/data-driven-form';
+import { API } from '../../http_api';
+import miqRedirectBack from "../../helpers/miq-redirect-back";
+
+const RegionForm = ({ id, maxDescLen }) => {
+  const [description, setDescription] = useState({});
+
+  useEffect(() => {
+    miqSparkleOn();
+    API.get(`/api/regions/${id}?attributes=description`)
+      .then(({ description }) => setDescription({ description }))
```

Comment:
`description = description` instead of `description = { description: description }`

```suggestion
      .then(({ description }) => setDescription(description))
```